en/config/actuators.md

```diff
@@ -227,6 +228,13 @@ In summary:
 - **Mixed Control Surfaces:** Upwards/rightwards movement is positive (as above).
   Includes V-Tail etc.
 
+::: tip
+Control surfaces that can only deflect in one direction from the neutral point include: Airbrakes, Spoiler, and Flaps.
+
+For these a positive input is always deflection from the neutral (0: no effect, 1: full effect), irrespective of the direction that the control itself moves.
+These do not respond to negative input.
+:::
+
```

Comment:
FYI @sfuhrer This is new text for the convention. This is intended to make it clear what controls the convention applies to, and that Airbrakes, spoilers, flaps are different.

Reply:
makes sense, looks good